plugins/star-rating/frontend/public/javascripts/countly.views.js

```diff
@@ -963,7 +963,7 @@
                 ];
             },
             ratingRate: function() {
-                var timesShown = this.widget.timesShown === 0 ? 1 : this.widget.timesShown;
+                var timesShown = this.widget.timesShown === 0 ? undefined : this.widget.timesShown;
```

Comment:
I dont think this key should be undefined.
@abhisheksingla02 can you please suggest changes?

Reply:
Here instead of making this key undefined , we should check why ratingsCount is not correct.

Reply:
by the formula logic, I chose to make it undefined so that the default value (0) can be returned by the formula,
but yes ofcourse timesShown cannot be zero if ratingCount is not zero.
Will look into the populator script.